Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM metabrainz/consul-template-base:v0.18.5-1
+
+ARG BUILD_DEPS=" \
+    build-essential \
+    git"
+
+ARG RUN_DEPS=" \
+    nodejs"
+
+RUN curl -sL https://deb.nodesource.com/setup_8.x | bash -
+RUN apt-get update
+RUN apt-get install --no-install-suggests --no-install-recommends -y \
+        $BUILD_DEPS \
+        $RUN_DEPS
+RUN rm -rf /var/lib/apt/lists/*
```

Comment:
You should put the `apt-get update` and `apt-get install` in the same `RUN` statement, because each `RUN` creates an intermediary (cached) image layer. So if you change `$RUN_DEPS`, it will use the old, cached `apt-get update` layer, which may be outdated at that point.

Reply:
I would also combine the `rm -rf /var/lib/apt/lists/*` into that same `RUN` statement too, since it's also good to reduce unnecessary image layers where they don't need to be cached separately (say because they are expensive).

Reply:
Good to know, will address.